src/test/java/core/basesyntax/AddFruitTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.AddFruit;
+import core.basesyntax.Fruit;
+import core.basesyntax.Transaction;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
```

Comment:
fix import statements order where needed
and don't use *

Reply:
done